src/linear_algebra/affine_space/combination.lean

```diff
@@ -127,6 +128,14 @@ begin
   exact set.sum_indicator_subset_of_eq_zero w (λ i wi, wi • (p i -ᵥ b : V)) h (λ i, zero_smul k _)
 end
 
+/-- A weighted sum combined with an embedding. -/
```

Comment:
Could you expand the doc-string? It doesn't seem to mention the conclusion.

Reply:
Several similar instances below.

Reply:
(Perhaps a good goal is for each doc-string to have a verb!)

Reply:
I've expanded these doc strings.